2-how-node-works/starter/events.js

```diff
@@ -0,0 +1,42 @@
+const EventEmitter = require(`events`);
+const http = require(`http`);
+
+class Sales extends EventEmitter {
+  constructor() {
+    super();
+  }
+}
+
+const myEmitter = new Sales();
+
+myEmitter.on(`newSale`, () => {
+  console.log(`There was a new sale`);
+}); // this is a listener
+myEmitter.on(`newSale`, () => {
+  console.log(`Costumer name: Vitor`);
+}); // this is a another listener (can have more than 1 for the same event)
+
+myEmitter.on(`newSale`, (stock) => {
+  console.log(`There are now ${stock} items left in stock.`);
+});
+myEmitter.emit("newSale", 9); // this is the emitter
+
+///////////////SEPARATOR////////////////////////////////////////////////////////////////
```

Comment:
Can you consider breaking the code of this file into 2 files: events_sales and events_server?